go.mod

```diff
@@ -1,9 +1,79 @@
 module github.com/grafana/mqtt-datasource
 
-go 1.16
+go 1.19
+
+replace github.com/grafana/grafana-plugin-sdk-go => /home/todd/grafana-plugin-sdk-go
```

Comment:
^^^. remove before commit

Reply:
removed in [88e5496](https://github.com/grafana/mqtt-datasource/pull/55/commits/88e5496ff1eb9cfdee2b3e6f05a8a75eadcc613c)